src/main/java/net/earthmc/emcapi/util/EndpointUtils.java

```diff
@@ -172,4 +175,35 @@ public static JsonObject getQuarterObject(Quarter quarter) {
 
         return jsonObject;
     }
+
+    public static JsonArray getOnlinePlayerArray(List<Player> players) {
+        JsonArray jsonArray = new JsonArray();
```

Comment:
Instead of directly doing an array we could do

```json
{
	"count": 123,
	"players": []
}
```

so that there's still an exact count available without opted out players.